src/main.cpp

```diff
@@ -2701,11 +2697,10 @@ bool AcceptBlock(CBlock& block, CValidationState& state, CBlockIndex** ppindex,
     return true;
 }
 
-static bool IsSuperMajority(int minVersion, const CBlockIndex* pstart, unsigned int nRequired)
+static bool IsSuperMajority(int minVersion, const CBlockIndex* pstart, unsigned nRequired, const Consensus::Params& consensusParams)
```

Comment:
This is a very minor nit, feel free to ignore.

I mean not taking Params, but do taking both the number to check and the
number required. It's a bit ugly to pass both params and something
extracted from params, imho.